Library/Tracking/KSRAnalytics.swift

```diff
@@ -553,12 +566,16 @@ public final class KSRAnalytics {
   // MARK: - Application Lifecycle
 
   public func trackTabBarClicked(_ tabBarItemLabel: TabBarItemLabel) {
-    let properties = contextProperties(tabBarLabel: tabBarItemLabel)
-
-    self.track(
-      event: ApprovedEvent.tabBarClicked.rawValue,
-      properties: properties
-    )
+    switch tabBarItemLabel {
+    case .search:
+      self.trackProjectSearchButtonClicked()
```

Comment:
I think it's ok to drop the logic directly in here since we are only using the method in one place

Reply:
I understand, I usually have a preference of moving any logic more than 4 lines in a switch case to a new function, just for readability. I have updated this regardless.